### PR TITLE
Vish

### PR DESCRIPTION
**Pull request by vishwa89**
Homepage changes

### CODE REVIEW

**Comment by AlexBulankou** [comment]
I ran based on latest in your branch and these are the comments on the visuals:
1. Menu is unstyled:
image
2. Dropdown is unstyled: 
image    
3. No social icons or "feedback" icon on footer
4. Logo overlaps content in desktop/tablet orientation when scrolling down:
image

5. This text is using different font size and line height compared to PSD. Please adjust:
image

I will now add the comments for the changes.

File: .gitignore [hunk 1]
```diff
@@ -7,4 +7,17 @@ node_modules
 stylecop.cache
 
 # CSHTML files should be generated
-*cshtml
\ No newline at end of file
+*cshtml
+src/cordova/www/images/v1/images/logo2b.png
+
+src/.DS_Store
```
Comment (AlexBulankou) [inline .gitignore]: just add .DS_Store to prevent these from being added anywhere

File: .gitignore [hunk 2]
```diff
@@ -7,4 +7,17 @@ node_modules
...
+
+src/.DS_Store
+
+src/.DS_Store
+
+src/client/.DS_Store
+
+src/client/content/v1/images/logo2-b.png
```
Comment (AlexBulankou) [inline .gitignore]: why do we need this entry?

File: .gitignore [hunk 3]
```diff
@@ -7,4 +7,17 @@ node_modules
 stylecop.cache
 
 # CSHTML files should be generated
-*cshtml
\ No newline at end of file
+*cshtml
+src/cordova/www/images/v1/images/logo2b.png
```
Comment (AlexBulankou) [inline .gitignore]: There shouldn't be any images under src/cordova/www/images/v1, all images should be under src/cordova/www/images/*.*. So instead of this entry, can you add src/cordova/www/images/v1/**/* if you're getting them accidentally copied under v1 folder (not sure why?)

File: 04e133c7-7fdc-46d8-8b72-29cc72c1e0ad.css [hunk 4]
```diff
@@ -0,0 +1,2687 @@
+/* Float Clearing
```
Comment (AlexBulankou) [inline 04e133c7-7fdc-46d8-8b72-29cc72c1e0ad.css]: Can you remove all these <guid>.css files from under root directory

File: src/client/content/styles_delayed.css [hunk 5]
```diff
@@ -3,25 +3,34 @@
...
+ /* background-image: url('v1/images/main_image2.jpg');*/
+        background-image:none;
+        background-color: #32394c;
 }
 
 .sbs-cdb .ui-overlay-a, 
 .sbs-cdb .ui-page-theme-a, 
 .sbs-cdb .ui-page-theme-a .ui-panel-wrapper {
-    background-image: url('../images/main_image_mobile2.jpg');
+ /* background-image: url('v1/images/main_image2.jpg');*/
```
Comment (AlexBulankou) [inline src/client/content/styles_delayed.css]: Don't put images under v1 directory for styles prefixed with .sbs-cdb. Instead they should be under ../images/, just like existing images

File: src/client/content/styles_delayed.css [hunk 6]
```diff
@@ -3,25 +3,34 @@
...
+        background-image:none;
+        background-color: #32394c;
+
     }
 
     .sbs-cdb .ui-overlay-a, 
     .sbs-cdb .ui-page-theme-a, 
     .sbs-cdb .ui-page-theme-a .ui-panel-wrapper {
-        background-image: url('../images/main_image2.jpg');
+        /* background-image: url('v1/images/main_image2.jpg');*/
```
Comment (AlexBulankou) [inline src/client/content/styles_delayed.css]: same image comment as above

File: src/client/content/v1/images/svg/sprites.svg [hunk 7]
```diff
@@ -4,7 +4,11 @@
 <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="176" height="32" viewBox="0 0 176 32">
 <defs>
 <g id="icon-fb">
+<<<<<<< HEAD
```
Comment (AlexBulankou) [inline src/client/content/v1/images/svg/sprites.svg]: These are GIT merge markings.  Please clean them up. In general, please check the files before checking them in.

File: src/client/content/v1/images/svg/sprites.svg [hunk 8]
```diff
@@ -4,7 +4,11 @@
...
   <path style="fill:#EB65A0;" d="M243.196,0C108.891,0,0,108.891,0,243.196s108.891,243.196,243.196,243.196
+=======
+  <path  d="M243.196,0C108.891,0,0,108.891,0,243.196s108.891,243.196,243.196,243.196
+>>>>>>> master
```
Comment (AlexBulankou) [inline src/client/content/v1/images/svg/sprites.svg]: same here

File: src/client/content/v1/images/svg/sprites.svg [hunk 9]
```diff
@@ -26,7 +30,11 @@
 			C319.992,397.021,305.666,385.282,272.59,361.775z" />
 </g>
 <g id="icon-ln">
+<<<<<<< HEAD
```
Comment (AlexBulankou) [inline src/client/content/v1/images/svg/sprites.svg]: same here

File: src/client/content/v1/images/svg/sprites.svg [hunk 10]
```diff
@@ -26,7 +30,11 @@
...
   <path style="fill:#EB65A0;" d="M243.196,0C108.891,0,0,108.891,0,243.196s108.891,243.196,243.196,243.196
+=======
+  <path  d="M243.196,0C108.891,0,0,108.891,0,243.196s108.891,243.196,243.196,243.196
+>>>>>>> master
```
Comment (AlexBulankou) [inline src/client/content/v1/images/svg/sprites.svg]: same here

File: src/client/html/partials/footer.handlebars [hunk 11]
```diff
@@ -1,38 +1,31 @@
 <div data-role="footer" data-position="fixed" class="jqm-footer" data-tap-toggle="false">
-    {{#if renderLetsGoButton}}
```
Comment (AlexBulankou) [inline src/client/html/partials/footer.handlebars]: why did you remove this section?

**Comment by AlexBulankou** [comment]
More general comments:
1. Remove all .DS_Store files
2. Remove src/client/content/v1/images/logo2-b.png
3. Remove src/cordova/www/images/logo2b.png
4. Don't add any files under src/cordova/www/images/v1/. Instead add them under src/cordova/www/images/ only

File: .gitignore (hunk 3 above)
Comment (vishwa89) [inline .gitignore]: ok

File: src/client/content/styles_delayed.css (hunk 5 above)
Comment (vishwa89) [inline src/client/content/styles_delayed.css]: This image was already there and added by you i just commented it.

File: src/client/content/styles_delayed.css (hunk 6 above)
Comment (vishwa89) [inline src/client/content/styles_delayed.css]: Same

File: src/client/html/partials/footer.handlebars (hunk 11 above)
Comment (vishwa89) [inline src/client/html/partials/footer.handlebars]: It was not there in the psd.

File: .gitignore (hunk 3 above)
Comment (vishwa89) [inline .gitignore]: This is also the logo was already present there did not add.

File: src/client/content/styles_delayed.css (hunk 5 above)
Comment (AlexBulankou) [inline src/client/content/styles_delayed.css]: Yes, sorry, please ignore this comment.

File: src/client/html/partials/footer.handlebars (hunk 11 above)
Comment (AlexBulankou) [inline src/client/html/partials/footer.handlebars]: It is needed in the footer for the route page

**Comment by AlexBulankou** [comment]
All looks good visually, just remove .DS_Store files and it will be ready to merge.

**Comment by AlexBulankou** [comment]
Merged into a branch
